core/templates/domain/topic/subtopic-page.model.ts

```diff
@@ -32,14 +32,14 @@ export interface SubtopicPageBackendDict {
 
 export class SubtopicPage {
   constructor(
-    private id: string,
-    private topicId: string,
-    private pageContents: SubtopicPageContents,
+    private id: string | null,
+    private topicId: string | null,
+    private pageContents: SubtopicPageContents | null,
```

Comment:
ditto as above

Reply:
Done PTAL is it okay

Reply:
Can we refactor the code so that the initialization with `null` is not needed (at least for some fields) and we fill the fields directly?

Reply:
completely removing createInterstitialSubtopicPage along with it's tests and assigning _subtopicPage as nothing, leads to too many errors in `topic-editor-services.ts`.
Please help me for further progress.
is it okay for now to add TODO comment and cover that part later. 
Thanks

Reply:
Yep, you can add a TODO

Reply:
Done